### DISCUSSION TITLE
1.1.10

### DISCUSSION THREAD

**Discussion by coreybutler** [coreybutler/nvm-windows#873] [General]
This is primarily a bug fix release. 

Many users have requested UAC prompts be reinstated. NVM4W attempts to run commands without elevation. If this succeeds (i.e. developer mode is enabled and active), the process continues without prompts. If a command fails, an attempt to run it with elevated privileges will trigger the UAC prompt.

## What's Changed
* Update error message when missing version argument. Fixes #722. by @0liver in https://github.com/coreybutler/nvm-windows/pull/731
* Fix README.md incorrect element close by @WORMSS in https://github.com/coreybutler/nvm-windows/pull/768
* Allow `nvm use` to be called with the part of the semver by @vladonemo in https://github.com/coreybutler/nvm-windows/pull/709
* Refactor/minor typo fixed in comments by @SayefReyadh in https://github.com/coreybutler/nvm-windows/pull/806
* Fix missing DisplayVersion registry key for nvm.iss by @awdr74100 in https://github.com/coreybutler/nvm-windows/pull/796
* Update inno setup config to correctly report version (fix #851) by @psnitil in https://github.com/coreybutler/nvm-windows/pull/868

## New Contributors
* @0liver made their first contribution in https://github.com/coreybutler/nvm-windows/pull/731
* @WORMSS made their first contribution in https://github.com/coreybutler/nvm-windows/pull/768
* @vladonemo made their first contribution in https://github.com/coreybutler/nvm-windows/pull/709
* @SayefReyadh made their first contribution in https://github.com/coreybutler/nvm-windows/pull/806
* @awdr74100 made their first contribution in https://github.com/coreybutler/nvm-windows/pull/796
* @psnitil made their first contribution in https://github.com/coreybutler/nvm-windows/pull/868

A few PR's were manually added, since merging them proved to be difficult after other changes. So, I'd like to give a special thanks to the following folks whose PR's were manually included (but not recognized in the changelog):
* @thengoster for https://github.com/coreybutler/nvm-windows/pull/784
* @craig-et for https://github.com/coreybutler/nvm-windows/pull/828 (which brought UAC prompts back)

**Full Changelog**: https://github.com/coreybutler/nvm-windows/compare/1.1.9...1.1.10

<hr /><em>This discussion was created from the release <a href='https://github.com/coreybutler/nvm-windows/releases/tag/1.1.10'>1.1.10</a>.</em>

**Comment by WORMSS** [comment]
❤️ 
Nothing more to say.

**Comment by kdpuvvadi** [comment]
will you be adding executable installer in the assets instead of zip for the latest release?

**Reply by coreybutler** [reply]
The zip contains the executable installer. It's always been packaged in a zip file because most company firewalls block the direct download of exe files.

**Reply by kdpuvvadi** [reply]
Yes, i get it for corporate env but we also use in the home. including exe would be nice for someone who needs it (it was included in the 1.1.9).

**Reply by coreybutler** [reply]
@kdpuvvadi I'm still not very clear why unzipping the file is a problem, but I added the executable to the 1.1.10 release for those who want to download it directly.

**Reply by kdpuvvadi** [reply]
unzipping a file is not that big of deal. We've this internal automation script that check for updates and installs them. we included nvm-windows before last release(1.1.9) and after 1.1.10, it broke on bunch of the boxes. So, We've uploaded extracted file temporarily on our file server. Our chief tech insisted on url from source. Then i've direct yesterday and added unzipping to script. 

We are migrating to winget internally. So, that won't be any issue as of today test repo is working. I've mentioned your rt project to concerned team, they are interested on that but might expect support.

**Reply by coreybutler** [reply]
@kdpuvvadi Runtime (rt) will have a core free version and a commercial supported version. The NVM4W winget installation is not an official installation, but Runtime will have an official winget installation option. 

If your team is interested (or anyone else reading this), the best way to be notified of Runtime updates/releases is to fill out the [survey](https://t.co/oGqQCM9FPx), which has a place for an email address to be notified. I'll also make an announcement on Twitter ([@goldglovecb](https://twitter.com/goldglovecb) and [@author_io](https://twitter.com/author_io)). I'm _hoping_ to have time to setup a formal website for Runtime over the holidays. In the meantime, the [Runtime Wiki Page](https://github.com/coreybutler/nvm-windows/wiki/Runtime) will be updated as progress is made.

**Comment by bhhoang818** [comment]
![Untitled](https://user-images.githubusercontent.com/88428487/201567666-15a4bc2f-559b-4f1f-a841-122fd4fd6586.png)
what was wrong

**Reply by coreybutler** [reply]
Did you download the installer or the updater from https://github.com/coreybutler/nvm-windows/releases/tag/1.1.10? Both of these installers produce version 1.1.10.

I have seen this with users who download from non-official sources. There was a problem with a version number in an older version of NVM4W (I think in 1.1.9), which was corrected within 2-3 days, but that was long enough for some of the mirrors to downloaded the "bad" version (they never updated).

**Comment by Jewgah** [comment]
Hi everyone, I've seen on a stackoverflow post ([link here](https://stackoverflow.com/questions/63697030/how-to-i-change-node-version-in-windows)) that node and npm need to be uninstalled manually before installing nvm, is that true or do latest versions take care of this issue ?

**Comment by alielmajdaoui** [comment]
2 security vendors and no sandboxes flagged this file as malicious!

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/32100313/203161967-73592236-e7e4-4a26-a02d-bc7bf77a5c69.png">

**Reply by 0liver** [reply]
There's already an issue tracking this: https://github.com/coreybutler/nvm-windows/issues/882.

**Reply by alielmajdaoui** [reply]
> There's already an issue tracking this: #882.

My bad! Thanks for the link

**Reply by coreybutler** [reply]
This resolved itself.

**Comment by 0liver** [comment]
How do we push that latest version to https://community.chocolatey.org/packages/nvm?

**Comment by Kristhian92** [comment]
fork/exec C:\Users\Usuario\AppData\Roaming\nvm\elevate.cmd:
Hello I get this error: The system cannot find the specified file.
How to solve it?

**Reply by coreybutler** [reply]
This file ships with the installer and the setup. If you installed it from another source, then I don't know why it would be missing. You can download it manually from https://github.com/coreybutler/nvm-windows/tree/master/bin.

**Reply by Kristhian92** [reply]
The file is fine, it appears in the folder but when I click on it nothing appears and when I open it in the terminal I get that error.
